### ISSUE TITLE
chat/forum

### ISSUE THREAD

**Issue by pythoneer** [areweguiyet/areweguiyet#8]
Do we want something like an official forum and/or chat? What other options do we have to bring people(users/developers/potential contributors) together to show of the current stage of their lib, app and to contact us. Making issues has a high barrier if you don't want plan to participate in a serious manner and just want to say hello and hang around.

**Comment by shivshank** [comment]
I think the main goal of a forum or chat would be:

Help crate authors discuss design choices with

**Comment by shivshank** [comment]
I think the main goal of a forum or chat would be: Help crate authors discuss design choices. Since each author or authors generally have other responsibilities (if they're working on an open source GUI), and already have their own chats for users and contributors, I don't think people want *another* place they have to check.

I also really like fschutt's post on [reddit](https://www.reddit.com/r/rust/comments/afhrru/rust2019_areweguiyet/edz6npl):

> In order to progress, you need to experiment, not just talk about things - I've seen this mistake so often, people discuss and discuss and discuss and in the end they have a wonderful hypothetical API that completely fails in practice.

Because of that, I don't really like forums because they encourage people to "blog" and write long posts instead of working on their projects! 😄 

Currently the places I know of for existing GUI discussion are:
 - Rust WG Gitter from last ~May(?) https://gitter.im/rust-gui-wg/Lobby
 - Unofficial Rust community discord #gui-and-ui channel: https://discord.gg/aVESxV8
 - and your discord (which is up to you if you want to make that "officially public"!)

And there's probably places on IRC. I think it would be best to choose one of these existing places and then link it from AWGY (if people are okay with it). And if people do blog and want to talk about what they're working on, they can either do it on AWGY already or link from this site with the newsfeed.

In short what I think:

For devs: AWGY helps them find other crates and crate authors
For users: AWGY helps them find crates they can use
For everyone: news feed helps them stay up to date with blogs and threads in the many places that exist
